x/spec/types/spec.go

```diff
@@ -122,6 +124,25 @@ func (spec Spec) ValidateSpec(maxCU uint64) (map[string]string, error) {
 				return details, fmt.Errorf("header names must be lower case %s", header.Name)
 			}
 		}
+
+		// get the spec's extension names list
+		extensionsNames := []string{}
```

Comment:
isnt it easier to use map?
and more efficient

Reply:
done. see https://github.com/lavanet/lava/commit/c53c81fbc93f42c33f1ac94671c3eb4cdb35cd07